events/views.py

```diff
@@ -2114,20 +2115,21 @@ def test_attendance(request, tid):
         test.save()
     except:
         raise PermissionDenied()
-    print((test.foss_id))
+    ta_status = request.GET.get('status',None)
     if request.method == 'POST':
         users = request.POST
         if users:
             #set all record to 0 if status = 1
             if 'submit-attendance' in users:
-                TestAttendance.objects.filter(test_id = tid, status = 1).update(status = 0)
+                if ta_status == None or ta_status == '1':
```

Comment:
```suggestion
                if ta_status is None or ta_status == '1':
```
use **is** instead of **==** for comparison with **None**

Reply:
Implemented the suggestion